### ISSUE TITLE
🐋 setup scripts fail trying to chown folders that are mounted by docker 

### ISSUE THREAD

**Issue by gordallott** [automatic-ripping-machine/automatic-ripping-machine#822]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Does this issue exist in the latest version?

- [X] I'm using the latest release

### Describe the bug?

the [arm_user_files_setup.sh](https://github.com/automatic-ripping-machine/automatic-ripping-machine/blob/main/scripts/docker/runit/arm_user_files_setup.sh#L46) script unnecessarily tries to chown directories, which will do nothing useful on most installs, and cause issues with anyone who has setup their nas to not allow random services to chown directories in the nas shares

if you have NFS setup to force specific groups so that every service you have is happy and does not encounter errors as random docker containers all have their own users/groups, these chown calls will fail causing the entire script to fail, causing the docker image to never enter a started state 

### To Reproduce

1. setup NFS with a mapall option to map all users to a specific user 
2. try and launch the ARM docker container
3. observe failure 

### Built from source files ?

- [ ] I Built the image from source

### What architecture are you seeing the problem on?

arm64

### Expected behavior?

the script should not attempt to chown folders that are expected to be mounted, it's always going to end in failure as different systems have different UAC requirements, it should just live within the defined permissions of the folders that are mounted. 

### Relevant log output

```shell
*** Running /etc/my_init.d/arm_user_files_setup.sh...
Updating arm group id 1000 to default (1000)...
usermod: no changes
Updating arm group id from 1000 to 1003...
Adding arm user to 'render' group
chown: changing ownership of '/home/arm/media': Operation not permitted
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by shitwolfymakes** [comment]
Thanks for pointing this out, I can see why it would be an issue. We do this to ensure that the files written by the container are not owned by root, otherwise you'd have to use sudo for every operation.

ARM specifically needs to be added to the render group, but that is something you'll have to do on the hosts end.

**Comment by TheRouce** [comment]
Is there any update on this? 

The arm user on my host machine has all the privileges it needs to read and write to the nfs share but not to chown.

**Comment by koeberlue** [comment]
I had the same problem when I was trying to use an SFTP mount. Here is what I did to make it run:

1. Inside my SFTP mount create a new folder, named `mymedia`
  1. So I had a folder `/path/to/mount/mymedia`
1.  `ls -n /path/to/mount/`
  1. This should print the user id and group id of the user owning `/path/to/mount/mymedia`.
  3. For me this was `1002` and `100` respectively
1. Edit the docker run command to use the new folder:

```
docker run -v "/path/to/mount/mymedia:/home/arm/media/" -e ARM_UID="1002" -e ARM_GID="100"
```  

The important change for me was to create the subfolder inside the mount, because this software would try to `chown` the folder `/path/to/mount/`  which would always fail.

**Comment by O-beron** [comment]
> Is there any update on this?

Any update ?

I'm merely a tinkerer and I get there's a reason why it's done this way, but doesn't that also cause a bunch of issues ? I don't want my network share to allow arbitrary programs to chown its directories, doesn't that go against good practices ?

**Comment by microtechno9000** [comment]
No update on this, but I can see the issue and it is a bit of a hangover from pre-docker. Will see what we can do

**Comment by microtechno9000** [comment]
Closed with #1053
